packages/embed/src/index.ts

```diff
@@ -130,6 +130,14 @@ export function setup(setupConfig: SetupConfig): EmbedInstance {
   // Attach elements to the DOM
   config.element.append(overlay, loader, frame)
 
+  // If the iframe failed to load or it took too much time, log a warning.
+  // This won't be displayed if the timer is cleared in frameReady.
+  const frameLoadWarn = setTimeout(() => {
+    console.warn(
+      'Loading Embed UI failed or took too long. Please check that the `gr4vyId` and `environment` values are correct.'
+    )
```

Comment:
@luca-gr4vy can we log the props passed to embed here as well? Much like we did in EmbedUI? No need to decrypt the token as this isn't an auth issue.

Reply:
ok, done